Source/common/MOLCodesignChecker.mm

```diff
@@ -92,14 +103,13 @@ - (instancetype)initWithSecStaticCodeRef:(SecStaticCodeRef)codeRef error:(NSErro
       if (infos && ![self allSigningInformationMatches:infos]) {
         status = errSecCSSignatureInvalid;
         NSString *description = @"Signing is not consistent for all architectures.";
-        cfError = (__bridge_retained CFErrorRef)[self errorWithCode:status description:description];
+        scopedError = ScopedCFError::Retain(
+            (__bridge CFErrorRef)[self errorWithCode:status description:description]);
       }
-    } else if (CFGetTypeID(codeRef) == SecCodeGetTypeID()) {
-      status = SecCodeCheckValidityWithErrors((SecCodeRef)codeRef, kSigningFlags, NULL, &cfError);
     }
 
     // Do not set _signingInformation or _certificates for universal binaries with signing issues.
-    NSError *err = CFBridgingRelease(cfError);
+    NSError *err = scopedError.Bridge<NSError *>();
```

Comment:
I'm concerned about lifetime here. This err is ultimately stored in the *error out-parameter. As an "owned" NSError that's fine, but here the underlying error is owned by the scopedError so when that goes out of scope won't it prematurely free?

Can we add a BridgeRelease method that uses CFBridgingRelease and then marks itself invalid?